roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -41,6 +41,8 @@ class RooFuncWrapper final : public RooAbsReal {
 
    void gradient(const double *x, double *g) const;
 
+   std::size_t getNumParams() { return _params.size(); }
```

Comment:
This method can be `const`, I think:
```suggestion
   std::size_t getNumParams() const { return _params.size(); }
```

Reply:
Yes, that would be good to change :+1: